app/main.py

```diff
@@ -1,8 +1,76 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        """
+        Приймає список автомобілів, миє лише ті, у яких
+        clean_mark менший за clean_power мийки
+        та повертає дохід, округлений до одного
+        десяткового знака. Автомобілі, які помито,
+        отримують новий clean_mark, рівний clean_power автомийки.
+        """
+        cars_for_washing = [
+            car
+            for car in cars
+            if car.clean_mark < self.clean_power
+        ]
+
+        for car in cars_for_washing:
+            self.wash_single_car(car)
+
+        return round(
+            sum(
+                [self.calculate_washing_price(car)
+                 for car in cars_for_washing]
+            ),
+            1
+        )
```

Comment:
You try to use list for summarize value, recommend debug this code locally

Reply:
Thanks

Reply:
> You try to use list for summarize value, recommend debug this code locally

The code has long been fixed and is working, it passes all the tests, but the task is displayed as incomplete on the mate platform.

Reply:
Try to send review again